models/model.js

```diff
@@ -50,4 +50,15 @@ exports.selectArticles = (topic) => {
   });
   };
      
+  exports.fetchArticleComments = (article_id) => {
+    return db
+      .query(`SELECT * FROM comments WHERE article_id = $1 order by created_at desc`, [article_id])
+      .then((result) => {
+        if (result["rowCount"] === 0) {

```

Comment:
This is the bit that might need changing, just because the article doesnt have comments doesn't mean that it needs to error, we might only want it to error if the article itself doesnt exist